examples/README.md

```diff
@@ -0,0 +1,44 @@
+# Examples
+
+![types.svg](./types.svg)
+
+Gantry supports `services` (oval shapes) and `steps` (box shapes).
+Services are docker containers executed like `docker-compose` is used, they
```

Comment:
This is already a good description (except for a grammar error) of the *how* but doesn't really say anything about the *why*. Here's my take that takes your facts for the *how* and mixes in the *why*.

"Services define docker containers which provide a continued service to other task in and outside the pipeline. They directly resemble the service concept from `docker-compose` and once started run concurrently to the rest of the pipeline including anything depending on the service.

Steps on the other hand run to completion and only then are their dependents executed. Note however, that independent steps are executed concurrently with each other. Steps are often used for tasks that produce a result that is needed by their dependents such as a download, creation of a database index or training of a machine learning model."

Also I think we should add this part to the main README

Reply:
Thank you for the text, I've added it to the main README now